apis/node/peer_count.yaml

```diff
@@ -0,0 +1,32 @@
+get:
+  operationId: getPeerCount
+  tags:
+    - Node
+  summary: Get peer count
+  description: Retrieves number of known peers.
+  responses:
+    "200":
+      description: Request successful
+      content:
+        application/json:
+          schema:
+            title: GetPeerCountResponse
+            type: object
+            properties:
+              data:
+                type: object
+                properties: 
+                  disconnected:
+                    type: number
+                    example: 12
```

Comment:
Do we use "number" anywhere else? or all just uint64?

Reply:
I think we should switch to uint64 to be consistent